app/app.js

```diff
@@ -34,7 +34,7 @@ openSansObserver.check().then(() => {
 });
 
 // Import the CSS reset, which HtmlWebpackPlugin transfers to the build folder
-import '../node_modules/sanitize.css/dist/sanitize.min.css';
+import '../node_modules/sanitize.css/sanitize.css';
```

Comment:
Ha, fair enough. For now, I just added a yet another blacklisted folder called `assets` where I define global style with imports (that is processed with post-css which allows this stuff). If the current approach works well - I am happy with leaving it as is. Just something to be aware of for the future.